src/components/Page/PageContent.module.css

```diff
@@ -0,0 +1,7 @@
+.page-content {
+  --component-page_content-vertical-padding: 2rem;
+  display: flex;
+  background-color: white;
```

Comment:
these values should use figma-tokens

Reply:
It doesn't exist figma tokens for this component yet so we need to add it when they've been created.

Reply:
Should we create an issue for it so we dont forget it then?

Reply:
We actually discussed in standup today that after each design system component is merged and out on https://designsystem.altinn.studio/ we should assign the issue for each component back to the designer so they can do a design review. In the review they should add all the missing Figma Tokens and comment the names of the tokens on the issue.